nano/rpc_test/rpc.cpp

```diff
@@ -5613,6 +5608,7 @@ TEST (rpc, DISABLED_wallet_history)
 	request.put ("action", "wallet_history");
 	request.put ("wallet", node->wallets.items.begin ()->first.to_string ());
 	auto response (wait_response (system, rpc_ctx, request));
+	boost::property_tree::write_json (std::cout, response);
```

Comment:
Having logs from tests is great, but please only write to stdout if logging is enabled.

Reply:
No, was not meant to  be there. I will remove it.